cdap-distributions/src/COPYRIGHT/jsesc@2.5.2/COPYRIGHT

```diff
@@ -0,0 +1,20 @@
+Copyright Mathias Bynens <https://mathiasbynens.be/>
+
+Permission is hereby granted, free of charge, to any person obtaining
```

Comment:
Do we have approval for this license? It does not look like a standard one.

Reply:
Is it MIT?

Reply:
Listed as MIT: https://www.npmjs.com/package/jsesc